Schema/ids.xsd

```diff
@@ -260,7 +260,8 @@
 			<xs:enumeration value="IFCRELASSIGNSTOGROUP"/> 
 			<xs:enumeration value="IFCRELCONTAINEDINSPATIALSTRUCTURE"/> 
 			<xs:enumeration value="IFCRELNESTS"/> 
-			<xs:enumeration value="IFCRELVOIDSELEMENT IFCRELFILLSELEMENT"/> 
```

Comment:
This line is by design, we want to use both relations in this case, and it looks like this is a valid way to enable the transformation of this attribute to a list in future versions of IDS.